### PR TITLE
Feature/remove leader role

### PR DESCRIPTION
**Pull request by mikonieminen**
Remove leader role and related request paths.

### CODE REVIEW

**Comment by mikonieminen** [comment]
This can be reviewed, but not merged yet. First we need to merge #15 as this is build on top of it.

**Comment by mikonieminen** [comment]
This is ready for review.

File: migrations/00000000000000_diesel_initial_setup/up.sql [hunk 1]
```diff
@@ -50,7 +50,6 @@ CREATE TABLE IF NOT EXISTS transactions (
     signature BLOB NOT NULL,
     validated BOOLEAN NOT NULL,
     bundle_id CHAR(43),
-    sent_to_leader BOOLEAN NOT NULL,
```
Comment (ghost) [inline migrations/00000000000000_diesel_initial_setup/up.sql:53]: Since we don't have launched yet, ok to put in the initial migration

**Review by ghost — APPROVED** [review]

File: migrations/00000000000000_diesel_initial_setup/up.sql (hunk 1 above)
Comment (mikonieminen) [reply migrations/00000000000000_diesel_initial_setup/up.sql:53]: That was my thinking too.

**Review by joshbenaron — APPROVED** [review]
LGTM

**Pull request merged**